common/CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ pkg_check_modules (OPENSSL REQUIRED openssl>=1.1.0g)
 ADD_LIBRARY(${UNTRUSTED_LIB_NAME} STATIC ${PROJECT_HEADERS} ${PROJECT_SOURCES} packages/block_store/block_store.cpp)
 
 TARGET_INCLUDE_DIRECTORIES(${UNTRUSTED_LIB_NAME} PRIVATE ${COMMON_PRIVATE_INCLUDE_DIRS})
+TARGET_INCLUDE_DIRECTORIES(${UNTRUSTED_LIB_NAME} PRIVATE "../eservice/pdo/eservice/enclave/enclave"})
```

Comment:
The common directory should not depend on anything outside of it IMO.

Reply:
fixed